src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1249,7 +1249,22 @@ internal WorkerThread WorkerThread
             get { return _worker; }
         }
 
-        internal string EscapePath(string path, bool ignoreSpaces = false)
+        internal string EscapePath(string path)
```

Comment:
>internal [](start = 8, length = 8)

I might suggest adding a comment to try and explain when to use this method and when to use EscapeSymbolPath. Might be worth renaming it to 'EscapeTargetMachinePath' also

```
/// <summary>
/// This method should be used to prepare file paths that are used are the target machine. For GDB Server scenarios, this means wherever gdbserver is being run.
/// </summary>
```
And on EscapeSymbolPath, maybe:
```
/// <summary>
/// This method should be used to escape paths that are used by GDB (and NOT gdbserver). Any path that gdbserver would use in remote server scenarios should use EscapeTargetMachinePath instead.
/// </summary>
```